lib/checking_account.rb

```diff
@@ -0,0 +1,42 @@
+require_relative 'account'
+
+module Bank
+  class CheckingAccount< Account
+    attr_accessor :id, :balance
```

Comment:
Turns out the `attr_accessor` methods are one of the things we get for free via inheritance, so this line is not needed.